NDB.Covid19/NDB.Covid19.Droid/Utils/LocalNotificationsManager.cs

```diff
@@ -36,28 +37,49 @@ public LocalNotificationsManager(Context context = null)
                 return;
             }
 
-            CreateChannel();
+            CreateChannels();
         }
 
-        private void CreateChannel()
+        private void CreateChannels()
         {
-            string channelName = NotificationContext.Resources.GetString(Resource.String.channel_name);
-            string channelDescription = NotificationContext.Resources.GetString(Resource.String.channel_description);
-            NotificationImportance importance = NotificationImportance.High;
+            NotificationChannel exposureChannel = new NotificationChannel(
+                _exposureChannelId,
+                NotificationChannelsViewModel.NOTIFICATION_CHANNEL_EXPOSURE_NAME,
+                NotificationImportance.High
+            )
+            {
+                Description = NotificationChannelsViewModel.NOTIFICATION_CHANNEL_EXPOSURE_DESCRIPTION,
+            };
+
+            exposureChannel.SetShowBadge(true);
```

Comment:
not all channels should show badges. it does not make sense for background channel

Reply:
sorry ignore